### ISSUE TITLE
Current uchiwa Dockerfile, bunch of 404's

### ISSUE THREAD

**Issue by guitmz** [sensu/uchiwa#452]
Hello,

I came across a problem using the latest uchiwa image. Blank pages and a bunch of 404 errors. After some search, I've found this https://github.com/sensu/uchiwa/issues/391 and the solution in the first comment (from the guy who opened the issue) worked for me, despite it being a bit old and the second guy saying that there was already a patch for that.

Am I missing something?

Thanks

**Comment by palourde** [comment]
Unfortunately I'm unable to reproduce this issue using the latest `uchiwa/uchiwa` image. Could you provide me the Image ID you are using?

Thanks

**Comment by palourde** [comment]
I'm closing this issue since we didn't received any responses.

If you are still facing this problem, please let us know with a comment and we'll be happy to reopen the issue.

Thanks!

**Comment by cjchand** [comment]
Let me know if you want me to open (and immediately close :) ) a separate issue, but came across a similar issue to this one while trying to roll my own Docker image.

tl;dr: Use the `-p` option to explicitly specify the full path to the public directory, like so:
`/opt/uchiwa/bin/uchiwa -c /etc/uchiwa/uchiwa.json -p /opt/uchiwa/src/public`

Longer version: I was building on the canned centos7 image and yum installing uchiwa (from the sensu repo), which resulted in 404s.

I suspect it's due to a similar issue reported by the OP and in #391: The uchiwa executable is looking for public in the "wrong" place.

Hope that helps someone in the future.
